bigbluebutton-tests/playwright/connectionFailure/connectionFailure.spec.js

```diff
@@ -0,0 +1,53 @@
+const { test, devices } = require('@playwright/test');
+const { ScreenShare, MultiUserScreenShare } = require('../screenshare/screenshare');
+const { sleep } = require('../core/helpers');
+const e = require('../core/elements');
+const { getCurrentTCPSessions, killTCPSessions } = require('./util');
+const notificationsUtil = require('../notifications/util');
+const deepEqual = require('deep-equal');
+
+test.describe.parallel('Connection failure', () => {
+  // https://docs.bigbluebutton.org/2.6/release-tests.html#sharing-screen-in-full-screen-mode-automated
+  test('Screen sharer', async ({ browser, browserName, page }) => {
+    test.skip(browserName === 'firefox' && process.env.DISPLAY === undefined,
+              "Screenshare tests not able in Firefox browser without desktop");
+    const screenshare = new ScreenShare(browser, page);
+    await screenshare.init(true, true);
+    await screenshare.startSharing();
+
+    await killTCPSessions(await getCurrentTCPSessions());
+
+    // I'd like to do this:
+    // await notificationsUtil.checkNotificationText(screenshare, "Code 1101. Try sharing the screen again.");
+
+    // but that code only checks the first toast, and there's a bunch of toasts that show up
+    // on this test.  So instead I use xpath, like this: (should we do it this way in checkNotificationText?)
```

Comment:
Good point. Doesn't seem to be a practical issue for `checkNotificationText`, but adjusting it the same way could improve the stability of `checkNotificationText`, I think.